app/views/admin/merchants/edit.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= form_with(url: admin_merchant_path(@merchant.id), method: :patch) do |form| %>
+  <%= form.label :name %>
+  <%= form.text_field :name, value: @merchant.name %>
```

Comment:
Nice job making sure the name shows up in the field.  We'll need to add this to some other forms for the merchant user stories portion later.